From 99ac549e00ed740e6fed5c0fc8457964f68e799e Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@xilinx.com>
Date: Thu, 7 Mar 2019 05:41:10 -0800
Subject: [PATCH 0554/1852] v4l: xilinx: multi-scaler: Fix precedence error

commit 7fc077cca20786ee14309a40720bc89cf5c17983 from
https://github.com/Xilinx/linux-xlnx.git

Fix precedence issues reported by checkpatch

Checkpatch reports :
$./scripts/checkpatch.pl --max-line-length 120 -f --strict
drivers/media/platform/xilinx/xilinx-multi-scaler.c

	CHECK: Macro argument 'x' may be better as '(x)' to avoid
precedence issues
	#75: FILE:
drivers/media/platform/xilinx/xilinx-multi-scaler.c:75:
	+#define XM2MSC_CHAN_REGS_START(x)      (0x100 + 0x200 * x)

Signed-off-by: Suresh Gupta <suresh.gupta@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-multi-scaler.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-multi-scaler.c b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
index fc05f6e0dd86..71af3a999e98 100644
--- a/drivers/media/platform/xilinx/xilinx-multi-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
@@ -72,7 +72,7 @@
 #define XM2MVSC_HFLTCOEFF_L	0x2800
 #define XM2MVSC_HFLTCOEFF(x)	(XM2MVSC_HFLTCOEFF_L + 0x2000 * (x))
 
-#define XM2MSC_CHAN_REGS_START(x)	(0x100 + 0x200 * x)
+#define XM2MSC_CHAN_REGS_START(x)	(0x100 + 0x200 * (x))
 
 /*
  * IP has reserved area between XM2MSC_DSTIMGBUF0 and
-- 
2.31.1

